tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -841,8 +841,11 @@ def _cloud_storage_no_new_errors(self, redpanda, logger=None):
         return increase == 0
 
     def stage_block_s3(self):
-        self.logger.info(f"Getting the first 100 segments into the cloud")
-        wait_until(lambda: nodes_report_cloud_segments(self.redpanda, 100),
+        self.logger.info(
+            f"Getting the first 100 segments into the cloud for specific topic"
```

Comment:
this f-string doesn't contain any interpolated values. did you intend to add some extra context to it?

Reply:
Good catch, it was like this before, I just modified it a bit. For now should be good